blocks/library/gallery/block.scss

```diff
@@ -0,0 +1,39 @@
+
+.blocks-gallery {
```

Comment:
@youknowriad let's figure out how to get these classes on the `edit` side of things.